bigtable/beam/workload-generator/src/test/java/bigtable/WorkloadGeneratorTest.java

```diff
@@ -124,16 +124,12 @@ public void testGenerateWorkload() {
 
     Pipeline p = Pipeline.create(options);
 
-    CloudBigtableTableConfiguration bigtableTableConfig =
-        new CloudBigtableTableConfiguration.Builder()
-            .withProjectId(options.getProject())
-            .withInstanceId(options.getBigtableInstanceId())
-            .withTableId(options.getBigtableTableId())
-            .build();
+    BigtableDataClient bigtableDataClient = BigtableDataClient.create(options.getProject(),
```

Comment:
something needs to close the client. I would recommend try with resources

Reply:
I actually don't even need this. I had it since I was initially passing the client in, but was struggling to figure out how to pass in a serialized client and instead decided to just pass in the project and instance ids as strings